trikot-viewmodels/databinding/api/databinding.api

```diff
@@ -1,11 +1,3 @@
-public abstract interface class androidx/databinding/DataBindingComponent {
```

Comment:
I'm not too sure why this is updated. Perhaps someone forgot to run apiDump after some changes ?